packages/style-loader/index.js

```diff
@@ -8,6 +8,7 @@ module.exports = (neutrino, opts = {}) => {
   const options = merge({
     test: neutrino.regexFromExtensions(['css']),
     ruleId: 'style',
+    oneOfId: 'normal',
```

Comment:
I'm wondering if we should just remove the configurability of `oneOfId`, `styleUseId`, `cssUseId`, `extractId` and so on, instead hardcoding the ids? (Leaving `ruleId` as the only configurable id)

The main reason to allow renaming is to avoid clashes, however that only really applies at the top level (eg if wanting to invoke `@neutrinojs/style-loader` twice with different settings).

Doing this would make the docs/options less daunting IMO.

Reply:
I agree. I'll axe em.

Reply:
The other option would be to leave it as is, but just remove them from the docs, making them "unsupported" options.

Reply:
I pushed a version that removes them from the docs but doesn't change the code (you technically _can_ still set those options).

Reply:
@edmorley Here's where I ended up: I removed oneOfId, styleUseId, cssUseId, modulesOneOfId as configurable options (as they're all nested under the style `ruleId` namespace anyhow.

I left `extractId` as we seem to do that with other added plugins, and I could see that being a conflict someone had to override.